system/jlib/jlog.hpp

```diff
@@ -236,6 +236,16 @@ typedef unsigned LogMsgDetail;
 #define DefaultDetail   100
 #define TopDetail (LogMsgDetail)-1
 
+constexpr int AllMsgThreshold         = 1;
+constexpr int FatalMsgThreshold       = 1;
+constexpr int ErrMsgThreshold         = 20-1;
```

Comment:
from offline conv. and your amended version:
```
constexpr int ErrMsgThreshold         = 10-1; //Used to declare categories which report Err messages (log level => 10)
constexpr int WarnMsgThreshold        = 20-1; //Used to declare categories which report Warn messages (log level => 20)
```

can these be ?:
```
constexpr int ErrMsgThreshold         = 10; //Used to declare categories which report Err messages (log level => 10 < 20)
constexpr int WarnMsgThreshold        = 20; //Used to declare categories which report Warn messages (log level => 20 < 30)
```

i.e. boundary start at 10,20,30 .. 

I think there was a suspicions that if starts at 10, and the maxDetails were [e.g.] set to 10, the jlog impl. would exclude them.